pkg/scheduler/api/devices/nvidia/vgpu/device_info.go

```diff
@@ -138,9 +135,6 @@ func (gs *GPUDevices) AddResource(pod *v1.Pod) {
 	podDev := decodePodDevices(ids)
 	for _, val := range podDev {
 		for _, deviceused := range val {
-			if gs == nil {
```

Comment:
Why these are removed?

Reply:
The nil judgment has already been processed on the outer layer, so there is no need to pay attention to whether it is nil inside the function

Reply:
Here will set nil node, so we should keep nil node check.
https://github.com/volcano-sh/volcano/blob/6b32f7bde9db76f40ddb19d253b06521ec7986ef/pkg/scheduler/cache/event_handlers.go#L214